lib/comment/comment.controllers.js

```diff
@@ -0,0 +1,141 @@
+const Joi = require('joi');
+
+module.exports = (models) => {
+
+  const Boom = require('boom');
+  const commentModels = require('./comment.models');
+  const accountLib = require('../account/account.controllers')(models).lib;
+  const postLib = require('../post/post.controllers')(models).lib;
+  const groupLib = require('../group/group.controllers')(models).lib;
+
+  return {
+    getComments: async function(request, h) {
+      let returnComments;
+      try {
+        returnComments = await getComments(request.query);
+      } catch (e) {
+        throw Boom.badImplementation('Error during getComments(request.query). ' + e);
+      }
+      return h
+        .response(returnComments);
+    },
+
+    getComment: async function(request, h) {
+      const returnComment = getComment(request.params.id);
```

Comment:
Ideally getComment would be wrapped in a `try` `catch` block. Also, ideally database calls wouldn't have their own `.then` `.catch` functions, because they should catch in the try catch blocks (so we can respond appropriately with a `throw`). (I know, you almost certainly based this off of code where I did this. I didn't realize this was sometimes problematic until later in the project.).